Sources/PackageDescription/Version+StringLiteralConvertible.swift

```diff
@@ -45,50 +42,61 @@ extension Version: ExpressibleByStringLiteral {
 }
 
 extension Version {
-
     /// Initializes a version struct with the provided version.
-    ///
-    /// - Parameters:
-    ///     - version: A version object to use for creating a new version struct.
+    /// - Parameter version: A version object to use for creating a new version struct.
     public init(_ version: Version) {
         major = version.major
         minor = version.minor
         patch = version.patch
         prereleaseIdentifiers = version.prereleaseIdentifiers
         buildMetadataIdentifiers = version.buildMetadataIdentifiers
     }
```

Comment:
@neonichu Please **don't** merge yet.

This initializer doesn't have any caller left after the change in `init(stringLiteral:)`. I don't remember why I didn't delete it.

Reply:
I deleted this initialiser and its tests, because it's really not called anywhere.

I think I might have confused myself when I was preparing for this PR initially, because in apple/swift-tools-support-core#214 an initialiser is deprecated but not removed, and I was working on both PR simultaneously.